### ISSUE TITLE
Crash on return from nether

### ISSUE THREAD

**Issue by Gebcrafter** [breakinblocks/nutsandboltstorqued#198]
[The Crash](https://pastebin.com/WTeXn2cc)
This happens on every return from the nether. It is because Mekanism is outdated.

**Comment by Saereth** [comment]
This is not a Mekanism versioning issue, we're a few minor (experimental) revisions behind but we're on the latest stable major revision. This appears to be a multipart issue and still has an open issue regarding it. https://github.com/amadornes/MCMultiPart/issues/71 The next pack update will have updated Mekanism but I cannot guarantee that will fix this issue as it appears to be specific to MCP and mekanism just happens to trigger it. 

I'll leave this issue open so we can continue to follow the progress of a fix for it.

**Comment by Gebcrafter** [comment]
Yeah, I was told that if updating Mekanism didnt fix it, to remove the offending entity with MCEdit. To the person who identified the crash for me on /r/feedthebeastcrashes wasn't too sure.

**Comment by Saereth** [comment]
There is also an option in forge.cfg to remove erroring tile entities on restart, that should fix it for you, I would not recommend leaving that option on after the fix though as its potentially problematic.
